slack-web.cabal

```diff
@@ -1,14 +1,14 @@
 cabal-version: 2.2
 name: slack-web
-version: 0.5.0.1
+version: 1.5.0.0
```

Comment:
why is it `1.5.0.0`? I would have thought it'd be `1.0.0.0`.

Reply:
Social reasons: I want to encourage people to do more releases rather than be like "oh it's 1.0". We managed to not release it for a few months already. Oops.

I would prefer to have something similar to Persistent in release schedule: every PR gets a release.